src/BenchmarksDriver/Program.cs

```diff
@@ -382,13 +385,8 @@ public static int Main(string[] args)
                 {
                     serverJob.Collect = true;
 
-                    serverJob.CollectArguments = collectTraceOption.Value();
-
-                    // Clear the arguments if the value is "on" as this is a marker for NoValue on the command parser
-                    if (serverJob.CollectArguments == "on")
-                    {
-                        serverJob.CollectArguments = "";
-                    }
+                    // Default to arguments which should be sufficient for collecting trace of default Plaintext run
+                    serverJob.CollectArguments = traceArgumentsOption.Value() ?? "BufferSize=1024";
```

Comment:
I think it should be appended, this way even if you set other options you get the default BufferSize.

Reply:
And put "BufferSize=1024" first so that `traceArgumentOption` can override the value.